programFlow.c

```diff
@@ -0,0 +1,81 @@
+//
+// Created by Ivan on 11/4/2019.
+//
+
```

Comment:
programFlow? name is not very suggestive and the functions are not grouped around a topic